### DISCUSSION TITLE
Can I deploy a business card website as part of the free plan?

### DISCUSSION THREAD

**Discussion by CharlesSquirel** [vercel/community#6205] [Help]
### Question

Can I deploy a business card website as part of the free plan?

I built a website in Next for a friend. This is a business card page with her details, company details, contact form and offer. There is no pricing details, purpose is only for contact with her. She is an English translator. What deployment plan do you recommend in such a case? Can I use the free form?

**Comment by amyegan** [comment]
Hey @CharlesSquirel. Personal contact info sites that don't charge for services are generally okay, but advertising the sale of a service could be considered [commercial usage](https://vercel.com/docs/limits/fair-use-guidelines#commercial-usage). Please take a look at our [Fair Use Guidelines](https://vercel.com/docs/limits/fair-use-guidelines) for more info on what is and is not allowed.
